src/pke/CMakeLists.txt

```diff
@@ -27,6 +27,11 @@ if( BUILD_SHARED )
 	install(TARGETS OPENFHEpke
 		EXPORT OpenFHETargets
 		DESTINATION lib)
+	target_include_directories(OPENFHEpke PUBLIC
```

Comment:
What feature is this going to provide to the library? Is this required for using  CMAKE_CURRENT_*?